scapy/layers/dot11.py

```diff
@@ -702,6 +702,14 @@ class Dot11(Packet):
                     FlagsField("FCfield", 0, 4,
                                ["pw-mgt", "MD", "protected", "order"]),
                     lambda pkt: (pkt.type, pkt.subtype) == (1, 6)
+                ),
+                (
+                    # FIXME: bw1/2/3 are a 3-bit field.
+                    # Unsure how to do this properly.
+                    FlagsField("FCfield", 0, 8,
+                               ["next_tbtt", "comp_ssid", "ano", "bw1",
+                                "bw2", "bw3", "security", "AP_PM"]),
```

Comment:
Good question; apart from doing something like that, I don't know (and that's not really great, so that's not a suggestion)...

@gpotter2 @guedou do you have a better idea?

```suggestion
                    FlagsField("FCfield2", 0, 2, ["security", "AP_PM"]),
                    BitField("FCfield_bw", 0, 3),
                    FlagsField("FCfield", 0, 3, ["next_tbtt", "comp_ssid", "ano"]),
```

Reply:
I think in that case I would split in in BitFields directly (so in this case 6 fields). That's what's usually done in Scapy (as opposed to FlagsFields)

Reply:
I'll try BitFields directly. Thanks @gpotter2 and @p-l- for your reviews.

Reply:
New change pushed which uses a combo of BitField and FlagsField. I am not willing to modify the existing FCField to be all BitFields as it may cause significant breakage of user scripts.